packages/typescriptlang-org/src/templates/documentation.tsx

```diff
@@ -159,7 +159,7 @@ const HandbookTemplate: React.FC<Props> = (props) => {
             </div>
             {showSidebar &&
               <aside className="handbook-toc">
-                <nav className={deprecationURL ? "deprecated" : ""}>
```

Comment:
Does removing this have any effect? It seems like what this is saying is that we should hide the ToC on deprecated pages?

Reply:
I don't really see a difference as such, and I'm not sure if we should actually hide the ToC on deprecated pages, but I can add it back if you think that's needed?